de.dlr.sc.virsat.model.extension.fdir/src/de/dlr/sc/virsat/model/extension/fdir/model/ReliabilityAnalysis.java

```diff
@@ -115,10 +117,15 @@ public Command perform(TransactionalEditingDomain ed, IProgressMonitor monitor)
 		if (subMonitor.isCanceled()) {
 			return UnexecutableCommand.INSTANCE;
 		}
-		subMonitor.split(1);
+		SubMonitor childMonitor1 = subMonitor.split(1);

```

Comment:
Should have a more descriptive name than "childMonitor1", e.g. "evaluationMonitor" or something like that.